addons/mass_mailing/models/mass_mailing.py

```diff
@@ -522,7 +522,7 @@ def _onchange_model_and_list(self):
         if self.mailing_model_name:
             if self.mailing_model_name == 'mail.mass_mailing.list':
                 if self.contact_list_ids:
-                    str_tuples += "('list_ids', 'in', {}),".format(','.join(str(id) for id in self.contact_list_ids.ids))
+                    str_tuples += "('list_ids', 'in', [{}]),".format(','.join(str(id) for id in self.contact_list_ids.ids))
```

Comment:
Couldn't we get rid of that "str_tuples" variable and the += ? It seems to me each case if independant and we could simply make mailing_domain in each case. IN that cas it would simply be something like
``
mailing_domain = [('list_ids', 'in', self.contact_list_ids.ids)]
...
self.mailing_domain = repr(mailing_domain)
``
cc @alexkuhn as I think you merged the mass mailing onboarding